src/sql/src/plan/statement/ddl.rs

```diff
@@ -1903,7 +1939,7 @@ fn kafka_sink_builder(
         Some(Format::Avro(AvroSchema::Csr {
             csr_connector:
                 CsrConnectorAvro {
-                    url,
+                    connector: CsrConnector::Inline { url: uri },
```

Comment:
nit: renaming this `uri` and then calling `uri.parse::<Url>()?` feels extra persnickety; I'd just call it a url

Reply:
Agreed, this was a transitionary naming thing that I I had caught all of.